spec.html

```diff
@@ -33221,7 +33221,7 @@
         <emu-alg>
           1. Assert: Both Type(_x_) and Type(_y_) is Number.
           1. If _comparefn_ is not *undefined*, then
-            1. Let _v_ be ? Call(_comparefn_, *undefined*, &laquo; _x_, _y_ &raquo;).
+            1. Let _v_ be ToNumber(? Call(_comparefn_, *undefined*, &laquo; _x_, _y_ &raquo;)).
```

Comment:
the `ToNumber` needs a `?` prefix as well, since it can throw

Reply:
Done